src/main/java/org/testng/ITestNGMethod.java

```diff
@@ -166,11 +166,10 @@
   void setInvocationCount(int count);
 
   /**
-   * @return the total number of thimes this method needs to be invoked, including possible
-   *         clones of this method - this is relevant when threadPoolSize is bigger than 1
-   *         where each clone of this method is only invoked once individually, i.e.
-   *         {@link org.testng.ITestNGMethod#getInvocationCount()} would always return 1.
+   * @deprecated Will always return 0

```

Comment:
Because it was introduced with https://github.com/cbeust/testng/commit/ce11604bda708dbd2c42435e77027d5c360c9fa9 but we don't use anymore.
And because it should include `getTotalParameterInvocationCount` but we are not able to determine it (with its iterator, data provider is lazy and we don't know the size)